framework/wazuh/core/cluster/dapi/dapi.py

```diff
@@ -256,6 +256,8 @@ def run_local():
                 data = await asyncio.wait_for(task, timeout=timeout)
```

Comment:
A question that comes to my mind. [Here](https://github.com/wazuh/wazuh/issues/9055#issuecomment-877231078) you say that the children inherit the space limitation from the parent process. But is the amount of memory used shared in any way? Or would it be possible to have, for example, a 2 GB limitation and run 20 simultaneous requests, each of which occupies 1 GB? Could we trick the limitation this way?

Reply:
The possible memory allocated limitation will be the same for each child process. 
With that sentence we can deduce that having a lot of childs running at the same time will trick the limitation. 

This cannot happen because the `ProcessPoolExecutor` creates by default a maximum number of processes equal to the number of processors in the machine. With this, we have that with a 2GB limitation, for instance, there could only be `2GB * n processors` of allocated memory at the same time by wazuh-apid + children.